### PR TITLE
Fix some typos.

### PR DESCRIPTION
**Pull request by viroulep**
Thanks to Chris Krueger and Tim McMahon who reported some typos.

### CODE REVIEW

**Comment by Laura-O** [comment]
Yes, let's collect all the typos here. That's probably easier than to make a PR for every single word.

File: wca-regulations.md [hunk 1]
```diff
@@ -163,7 +163,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 4d5) Skewb is scrambled starting with the white face (if not possible, then the lightest face) on top and the green face (if not possible, then the darkest adjacent face) on the front-left.
 - 4f) Competition scramble sequences must be generated using the current official version of the official WCA scramble program (available [via the WCA website](https://www.worldcubeassociation.org/regulations/scrambles/)).
 - 4g) After scrambling a puzzle, the scrambler must verify that the puzzle is scrambled correctly. If the puzzle state is wrong, the scrambler must correct it (e.g. by solving the puzzle and applying the scramble sequence again).
-    - 4g1) Exception: For the 6x6x6 Cube, 7x7x7 Cube, and Megaminx, it is not necessary to correct the scramble sequence, at the discretion of the WCA Delegate.
+    - 4g1) Exception: For the 6x6x6 Cube, 7x7x7 Cube, and Megaminx, it is not necessary to correct the puzzle state, at the discretion of the WCA Delegate.
```
Comment (Laura-O) [inline wca-regulations.md:166]: Isn't this the "scrambled state"?

**Comment by viroulep** [comment]
Ah sorry I did not clearly updated that I fixed the "be permitted" too :)

File: wca-regulations.md (hunk 1 above)
Comment (viroulep) [inline wca-regulations.md:166]: It could be, but "puzzle state" is used in 4g, so I figured I would use the same wording.

**Pull request merged**